Dockerfile.with-sidecar

```diff
@@ -0,0 +1,38 @@
+# Build Container
+FROM golang:1.13-buster AS build-env
+
+# Get argment
+ARG TOPOLVM_VERSION
+
+COPY . /workdir
+WORKDIR /workdir
+
+RUN make build TOPOLVM_VERSION=${TOPOLVM_VERSION}
+
+# TopoLVM container
+FROM ubuntu:18.04
```

Comment:
Please use ubuntu 20.04 as the PR that updates ubuntu has been merged recently.

https://github.com/topolvm/topolvm/blob/adcd4c9fd4ba26e24d1ee130a0e28646350ce7ab/Dockerfile#L13